tests/pgautofailover_utils.py

```diff
@@ -284,6 +304,10 @@ def create(self):
                              name="keeper init",
                              timeout=COMMAND_TIMEOUT)
 
+        if (self.get_state() == 'primary'):
+            super().create_authenticated_users()
```

Comment:
Reading the test cases more closely, I'm now thinking that:

  1. the method name sounds wrong, because it does not create any user, only adds passwords to them
  2. calling that automatically from the test _infrastructure_ looks like the wrong place to me now, I think we should expose the API and then call it in the test file where we pick the authMethod to be md5, that's where we want to install and control the passwords after all

Reply:
addressed both concerns
1 - removed the function, replaced with set_user_password function to set individual user password
2 - called the new function from test file

Reply:
I'm not sure I understand how that works wrt to setting the password both on the connection string and for the user on the target database...